ci-operator/step-registry/openshift-qe/orion/openshift-qe-orion-commands.sh

```
@@ -189,7 +189,12 @@ fi
 # pull_number input variable is required and
 # it must be set as $PULL_NUMBER OR 0 to get compared against periodic runs.
 pull_number='0'
-if [[ "${JOB_TYPE}" == "periodic" ]]; then
+if [[ "${ORION_PR_ANALYSIS:-false}" == "true" ]] && [[ "${JOB_TYPE}" == "presubmit" ]] && [[ -n "${PULL_NUMBER:-}" ]]; then
+    job_type="pull"
+    pull_number="${PULL_NUMBER}"
+    EXTRA_FLAGS+=" --pr-analysis"
+    EXTRA_FLAGS+=" --input-vars '{\"jobtype\":\"pull\",\"pull_number\":\"${PULL_NUMBER}\",\"organization\":\"${REPO_OWNER}\",\"repository\":\"${REPO_NAME}\"}'"
```

Shellcheck (0.11.0)

[warning] Quotes/backslashes will be treated literally. Use an array.

(SC2089)